backend/src-common/src/main/java/org/eclipse/sw360/datahandler/db/ComponentDatabaseHandler.java

```diff
@@ -3064,9 +3064,10 @@ public RequestStatus uploadSourceCodeAttachmentToReleases() {
         log.info(String.format("SRC Upload: Found %d components with VCS", components.size()));
 
         components.forEach(c -> {
-            String VCS = c.getVcs();
+            String VCS = c.getVcs().toLowerCase();
```

Comment:
@FarooqAftab you need to make the change on line 3070. Converting the entire vcs to lower case and then processing it might cause unwanted issues.